index.d.ts

```diff
@@ -81,7 +81,8 @@ export declare interface Tracer extends opentracing.Tracer {
    * If the `orphanable` option is set to false, the function will not be traced
    * unless there is already an active span or `childOf` option.
    */
-  trace<T> (name: string, fn: (span?: Span, fn?: (error?: Error) => any) => T): T;
+  trace<T> (name: string, fn: (span: Span) => T): T;
+  trace<T> (name: string, fn: (span: Span, done: (error?: Error) => string) => T): T;
   trace<T> (name: string, options: TraceOptions & SpanOptions, fn: (span?: Span, done?: (error?: Error) => string) => T): T;
```

Comment:
The return type of the `done` callback is/was listed as `any` (with the no-options signature) and `string` (with the other signature), but looking at the code it seems to be a `void` -- should that be corrected too?

Reply:
`tracer.trace` has a generic parameter `<T>` which matches the return type of `fn`; however, i'm referring to the `done` callback (which was - confusingly - also called `fn` in the current version of the code, though i have proposed to rename it).

Arrows pointing to the old and new callbacks
![image](https://user-images.githubusercontent.com/877396/212754638-039d0e31-1d3d-43d2-babc-e02d829ba9af.png)

Reply:
note the return type of `fn` remains as `T`
![image](https://user-images.githubusercontent.com/877396/212761216-427ca70e-105a-49ae-b70f-703a617004ca.png)